static/src/javascripts/projects/journalism/snippet-feedback.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+import mediator from 'lib/mediator';
+import ophan from 'ophan/ng';
+import detect from 'lib/detect';
+
+const SnippetFeedback = () => {
+    const snippets = document.querySelectorAll('.explainer-snippet--new');
+
+    [].forEach.call(snippets, snippet => {
+        const atomId = snippet.getAttribute('data-snippet-id');
+        const snippetType = snippet.getAttribute('data-snippet-type');
+        if (!atomId || !snippetType) {
+            return;
+        }
+        const component = `snippet_${snippetType}`;
+
+        // Callback for like/dislike
+        const onFeedback = e => {
+            const question = e.currentTarget;
+            const ack = snippet.querySelector('.explainer-snippet__ack');
+            if (!(question instanceof HTMLElement)) {
```

Comment:
I was looking for a cleaner way to do this, nice! I've been using ⊥ for a while with Flow but this use case is definitely the most useful one so far!